cvat/schema.yml

```diff
@@ -2271,10 +2338,10 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/AnnotationFileRequest'
+              $ref: '#/components/schemas/UploadedFileRequest'
```

Comment:
This change (and others like it) introduces a backwards incompatibility on the API level. Can we avoid it?

Reply:
> This change (and others like it) introduces a backwards incompatibility on the API level

Unfortunately, yes it is, but do you see any benefit in using different field names when we upload backups/datasets/annotations not via TUS?
```
POST /api/tasks/backup:  "task_file": "string"
POST /api/projects/backup: "project_file": "string"
POST /api/projects/id/dataset: "dataset_file": "string"
POST /api/tasks|jobs/id/annotations: "annotation_file": "string"
```
Why not just use a unified field? It should be obvious that e.g. POST /api/tasks/backup expects task backup file.